### ISSUE TITLE
Improve usability of creating new hard disk image

### ISSUE THREAD

**Issue by MBilderbeek** [openMSX/openMSX#1738]
Use case: create a new formatted hard disk image for quick use with e.g. the Sunrise IDE interface.

The manual has this use case described in terms of console commands:

> Start openMSX with the ide extension, then type in the console:
> 
> set power off
> diskmanipulator create /tmp/new-hd.dsk 32M 32M 32M
> hda /tmp/new-hd.dsk
> diskmanipulator import hda1 /home/david/msxdostools/
> diskmanipulator import hda2 /home/david/msxdemos/
> diskmanipulator import hda3 /home/david/msxdrawings/
> set power on

I was attempting to describe this scenario if it had to be executed with the GUI. But that wasn't as simple as I hoped:
1. You have to think up front to power off the MSX. That's not intuitive as a first step. So let's skip it now.
2. Open the Disk Manipulator window and create the image. You have to know up front which type. DOS2 should suffice for IDE without Nextor.
3. You can't use the Disk Manipulator now to directly copy files on these partitions... you first have to think of putting it in the running MSX with the Media -> Hard Disk A menu item. But that is disabled, so you must go to Machine -> Power to turn off the power.
4. Then back to Media -> Hard Disk A to browse to the just created image (first find it back.... browsing a lot)
5. Then you can go back to the Disk Manipulator and select the partitions and copy the files
6. Then you go back to Machine -> Power to turn on the power

In other words: the GUI doesn't help a lot to make this scenario easier.

**Comment by MBilderbeek** [comment]
@m9710797 I was pointed to the fact that in the OSD menu, you could directly change harddisk image, and the menu could would turn off the MSX for you to do so, first asking "Really power off to change HDD image?". If you confirm, it powers off, changes HDD image and powers back on. 
That we don't do that anymore in 20.0 is considered a regression by rolvandve (and I understand that): see https://www.msx.org/forum/msx-talk/openmsx/installing-a-hard-disk-on-openmsx-v200
